### PR TITLE
Reconsiders my trust in players in regards to VIP Missions, fixes admin mald

### PR DESCRIPTION
**Pull request by Tsar-Salat**
## About The Pull Request

Changes VIP Official's suit to a bulletproof vest instead of an armored coat

Changes Centcom Admiral to use centcom hat instead of red beret

## Why It's Good For The Game

Someone repeatedly gamed the VIP Centcom Officials cool coat I added in #11097, enough to get staff notice. I'm disappointed, but not suprised

For centcom admirals, admins didn't like red beret with their green drip

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
tweak: Swapped VIP centcom official's coat for a vest
tweak: Switched Centcom Admiral's red beret for a centcom caphat, per admin request
/:cl:

### CODE REVIEW

File: code/modules/clothing/outfits/standard.dm [hunk 1]
```diff
@@ -282,7 +282,7 @@
 	glasses = /obj/item/clothing/glasses/thermal/eyepatch
 	ears = /obj/item/radio/headset/headset_cent/commander
 	mask = /obj/item/clothing/mask/cigarette/cigar/havana
-	head = /obj/item/clothing/head/helmet/space/beret
+	head = /obj/item/clothing/head/hats/centhat
```
Comment (PowerfulBacon) [inline code/modules/clothing/outfits/standard.dm:285]: Keep the special space-proof armour for the admirals

**Comment by PowerfulBacon** [comment]
You shouldn't be neither surprised nor disappointed that players equip the most powerful armour in the game when given the opportunity. I didn't spot this in the original PR, the VIP target is just an official and not an admiral.

File: code/modules/clothing/outfits/standard.dm (hunk 1 above)
Comment (Tsar-Salat) [reply code/modules/clothing/outfits/standard.dm:285]: I mean, admins said they didn't like the look.

May I make a snowflake subtype of the hat that's space proof? Or is there a better way

Comment (PowerfulBacon) [reply code/modules/clothing/outfits/standard.dm:285]: I guess

**Review by PowerfulBacon — APPROVED** [review]

**Comment by PowerfulBacon** [comment]
whatever, we need this

**Pull request merged**